.github/workflows/consecutive.yml

```diff
@@ -267,20 +212,33 @@ jobs:
           ref: gh-pages
           path: gh-pages
 
-      # - name: Allure Report with history
-      #   # uses: simple-elf/allure-report-action@v1.5
-      #   uses: simple-elf/allure-report-action@master
-      #   if: always()
+      - name: Create a path for report
+        run: |
+          path=""
+          network="${{ env.NETWORK }}"
+          ref_name="${{ github.ref_name }}"
+          if [[ "$network" != "teststand2" ]]; then
+            path="$network"
+          fi
+          if [[ "$ref_name" != "develop" ]]; then
+            if [ ${#path} -gt 0 ]; then
+              path="$path/$ref_name"
+            else
+              path="$ref_name"
+            fi
+          fi
+          echo "ALLURE_SUBDIR=$path" >> $GITHUB_ENV
 
       - name: Allure Report action from marketplace
         uses: neonlabsorg/allure-report-action@master
         if: always()
         with:
           allure_results: allure-results
           gh_pages: gh-pages
+          subfolder: ${{ env.ALLURE_SUBDIR }}
           allure_report: allure-report
           allure_history: allure-history
-          keep_reports: 30
+          keep_reports: 50
```

Comment:
why do you need 50? isn't it working too slowly?

Reply:
No, this momen doesn't affect speed. I increase this for save same logs in branches.